src/test/ui/proc-macro/inner-attrs.rs

```diff
@@ -24,18 +24,30 @@ struct MyStruct {
 }
 
 fn bar() {
-    (#![print_target_and_args(fifth)] 1, 2);
+    (#![print_target_and_args(fifth)] 1, 2); //~ ERROR expected
```

Comment:
```suggestion
    (#![print_target_and_args(fifth)] 1, 2); //~ ERROR expected non-macro inner attribute, found attribute macro
```
Could you use more detailed messages after `//~ ERROR` annotations.